### DISCUSSION TITLE
page insert text bug

### DISCUSSION THREAD

**Discussion by hbh112233abc** [pymupdf/PyMuPDF#1889] [Q&A] [answered]
_**Please provide all mandatory information!**_

## Describe the bug (mandatory)
I find some chars insert the page can not show!
like Ⓐ,⑱

## To Reproduce (mandatory)
```
import fitz
doc = fitz.open()
txt = 'Ⓐ'
page = doc.new_page()
page.insert_text((100,100), txt,fontsize=30,fontfile="c:/Windows/Fonts/simhei.ttf")
doc.save('test.pdf')
```

## Expected behavior (optional)
I try set other fontfile,and change `encoding` param, but not effect

## Screenshots (optional)
![image](https://user-images.githubusercontent.com/7788164/185878084-829c34d7-1fd0-4c15-a200-baea0dd863fa.png)

## Your configuration (mandatory)
windows 10
3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] 
 win32 
 
PyMuPDF 1.20.0: Python bindings for the MuPDF 1.20.1 library.
Version date: 2022-06-27 00:00:01.
Built for Python 3.10 on win32 (64-bit).

**Comment by JorjMcKie** [comment]
This is not a bug, but a "question". So I transfer it to the **_Discussions_** tab for further handling.

**Comment by JorjMcKie** [comment, marked as answer]
You must of course always use a font containing the required symbols. This is a working example:
```python
doc=fitz.open()
page=doc.new_page()
text="text="Ⓐ ⑱ ➀ ➁ ➂ ➃ ➄ ➅ ➆ ➇ ➈ ➉"
point= (100, 100)
page.insert_font("F0",fontfile="noto-sans-symbols-regular.ttf")
page.insert_text(point, text, fontname="F0")
doc.save("x.pdf")
```

**Reply by hbh112233abc** [reply]
thanks
